api/app/src/main/kotlin/packit/controllers/UsersController.kt

```diff
@@ -0,0 +1,24 @@
+package packit.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import packit.model.dto.UserDto
+import packit.model.toDto
+import packit.service.UserService
+
+@Controller
+@PreAuthorize("hasAuthority('user.manage')")
+@RequestMapping("/users")
```

Comment:
heads up there is already a `UserController` so can add to that :smile:

Reply:
I saw that, but this is for `/users` plural rather than `/user` singular, so I thought maybe a separate controller would be in order. But was going to ask you about that, fine to combine them if you'd rather!

Reply:
yeah definitely just stick it in the existing controller. It was my fault when I made it must called it `user` instead of `users` by mistake.. because to get singular you would just do `/users/{id}`